.nvmrc

```diff
@@ -1 +1 @@
-v17.1.0
+v16.14.2
```

Comment:
I just wanted to align the node version, let me know if this is an issue

Reply:
We have an issue currently with NextJS themes kubernetes pods crashing on Node 16, so we rolled back the deployment version to v14. But for local development Node 16 should be fine I believe.